packages/react-components/src/AccessibilityScenarios/Input.stories.tsx

```diff
@@ -0,0 +1,209 @@
+import * as React from 'react';
+
+import { Input, InputOnChangeData } from '@fluentui/react-input';
+import { Label } from '@fluentui/react-label';
+import { Button } from '@fluentui/react-button';
+
+import { Scenario } from './utils';
+
+export const RegistrationFormInputsAccessibilityScenario = () => {
+  // eslint-disable-next-line
+  const validations: Record<string, any> = {
```

Comment:
Can you locally declare a type here instead of any? Like `type InputValidation = { type: React.InputHTMLAttributes['type']; regexes: RegExp[] }`

Also, the `validations` const would be more appropriate to declare at the top of the file IMO since it doesn't depend on anything inside the component.

Reply:
Good points. Fixed.